app/javascript/src/components/Dashboard/Contacts/Table/RowActions.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { MenuHorizontal } from "@bigbinary/neeto-icons";
+// @ts-ignore
+import { Dropdown } from "neetoui";
```

Comment:
https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663146331895639?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

Reply:
I did not add the relevant aliases. I will rectify this.